webpack.config.js

```diff
@@ -190,7 +128,6 @@ module.exports = {
                         options: {
                             sourceMap: !production,
                             sassOptions: {
-                                quietDeps: true,
```

Comment:
Was this a workaround for PF3? We have quietDeps in cockpit and cockpit-composer as well. Mind if you back that out and investigate separately?

Reply:
I'm not sure, everything compiled fine and looked fine so hmmm :eyes: